processing/src/main/java/io/druid/segment/IndexSpec.java

```diff
@@ -91,7 +94,8 @@ public IndexSpec()
   public IndexSpec(
       @JsonProperty("bitmap") BitmapSerdeFactory bitmapSerdeFactory,
       @JsonProperty("dimensionCompression") String dimensionCompression,
-      @JsonProperty("metricCompression") String metricCompression
+      @JsonProperty("metricCompression") String metricCompression,
+      @JsonProperty("metricLongEncoding") String metricLongEncoding
```

Comment:
this will require documentation updates.

In general we need docs for all these new compression methods